src/help/plugin.ts

```diff
@@ -149,19 +149,21 @@ function activate(app: JupyterLab, mainMenu: IMainMenu, palette: ICommandPalette
   // Handle state restoration.
   restorer.restore(tracker, {
     command,
-    args: widget => ({ isHidden: widget.isHidden, url: widget.url }),
-    name: widget => namespace
+    args: widget => ({ url: widget.url }),
```

Comment:
Thanks! Trying that now. Can you clarify how those keys get used to re-created the widget upon restoration? I tried to look around and find what keys were expected, but didn't get anywhere.